app/views/orga/teams/_form.html.slim

```diff
@@ -77,14 +77,26 @@
     = f.input :season_id, as: :select, collection: Season.order('name DESC') { |e| [s.name, s.id] }
 
   .page-header
-    h2 Conference preferences
+    h2 Conferences
+    h3 Conference preferences
```

Comment:
Since this is a heading, I'd suggest to capitalize it:
> Conference Preferences

_(same for the offers)_